aws/rds_cluster.go

```diff
@@ -0,0 +1,97 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"time"
+
+	awsgo "github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/gruntwork-io/cloud-nuke/logging"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+)
+
+func waitUntilRdsClusterDeleted(svc *rds.RDS, input *rds.DescribeDBClustersInput) error {
+	// wait up to 60 seconds
```

Comment:
I don't think 60 seconds is enough to wait for an RDS cluster to be deleted. This may be enough time for Aurora Serverless, but definitely not enough for regular Aurora. This should probably be more like 15 mins.

Reply:
✅